test-e2e/manager-invite.js

```diff
@@ -637,48 +637,41 @@ test('disconnect before invite accept', async (t) => {
 
 // TODO: It's not possible in e2e tests to disconnect peers at the right moment.
 // This test is flaky, and sometimes the project details are sent before the
-// peers disconnect, so skipping this test for now.
-test(
-  'disconnect before sending project join details',
-  { skip: true },
-  async (t) => {
-    const clock = FakeTimers.install({ shouldAdvanceTime: true })
-    t.after(() => clock.uninstall())
-    const [creator, joiner] = await createManagers(2, t)
-    const disconnectPeers = connectPeers([creator, joiner])
-    t.after(() => disconnectPeers())
-    await waitForPeers([creator, joiner])
-
-    const createdProjectId = await creator.createProject({ name: 'Mapeo' })
-    const creatorProject = await creator.getProject(createdProjectId)
-
-    const inviteReceivedPromise = pEvent(joiner.invite, 'invite-received')
-    const invitePromise = creatorProject.$member.invite(joiner.deviceId, {
-      roleId: MEMBER_ROLE_ID,
-    })
-    const assertInviteRejectsPromise = assert.rejects(
-      invitePromise,
-      {
-        name: 'PeerDisconnectedError',
-        message: 'Peer disconnected before sending project join details',
-      },
-      'invite promise rejects when peer disconnects'
-    )
-    const invite = await inviteReceivedPromise
-    const assertAcceptRejectsPromise = assert.rejects(
-      () => joiner.invite.accept(invite),
-      {
-        name: 'TimeoutError',
-        message: 'Timed out waiting for project details',
-      },
-      'accepting invite times out when project details are not received'
-    )
-    await disconnectPeers()
-    // Run down the timeout waiting for project details
-    clock.runAll()
-    await Promise.all([assertInviteRejectsPromise, assertAcceptRejectsPromise])
-  }
-)
+// peers disconnect, so can't expect specific errors.
+test('disconnect before sending project join details', async (t) => {
```

Comment:
if it's not already tested elsewhere, I think it would be helpful to extend this test to make sure that re-inviting can work after the failed attempt simulated here

Reply:
Good idea!

Reply:
Good catch, we got an error saying we can't assign the LEFT role to another device.

Reply:
Yeah looks like I need a new role